code/starfield/starfield.cpp

```diff
@@ -2301,9 +2301,7 @@ void stars_set_background_model(const char *model_name, const char *texture_name
 	if (Nmodel_num >= 0) {
 		model_page_in_textures(Nmodel_num);
 
-		if (model_get(Nmodel_num)->flags & PM_FLAG_HAS_INTRINSIC_MOTION) {
-			Nmodel_instance_num = model_create_instance(-1, Nmodel_num);
-		}
+		Nmodel_instance_num = model_create_instance(-1, Nmodel_num);
```

Comment:
Elsewhere I've tried to avoid creating model instances unless they are needed.  Since I assume the majority of skyboxes will not need instances, this seems inefficient.  On the other hand, it's just one model per mission and `polymodel_instance` is pretty lightweight.

Reply:
We'd have to pass the information that a submodel animation might exist to there, which isn't worth the effort for one measly model instance IMO

Reply:
Fair enough